# Enable/disable LiDAR motor
bool enable
---
bool success
string message
